processor/remotetapprocessor/processor.go

```diff
@@ -12,13 +12,15 @@ import (
 	"sync"
 	"time"
 
+	"go.uber.org/zap"
+	"golang.org/x/net/websocket"
```

Comment:
Probably will fail goimports?

Reply:
Lint failed as @atoulme pointed out:

```
remotetapprocessor/processor.go:15: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
level=info msg="File cache stats: 12 entries of total size 23.4KiB"
level=info msg="Memory: 21 samples, avg is 153.0MB, max is 289.6MB"
level=info msg="Execution took 1.91397649s"
	"go.uber.org/zap"
	"golang.org/x/net/websocket"
	"golang.org/x/time/rate"

remotetapprocessor/processor.go:23: File is not `gci`-ed with --skip-generated -s standard -s default -s prefix(github.com/open-telemetry/opentelemetry-collector-contrib) (gci)
	"go.opentelemetry.io/collector/processor"
make[2]: *** [../../Makefile.Common:199: lint] Error 1
```

Reply:
Thank you for your review. 
goimports format issue, fixed.